proto/src/determined/api/v1/trial.proto

```diff
@@ -797,7 +797,29 @@ message GetTrialSourceInfoMetricsByCheckpointRequest {
 }
 
 // Response for all metrics related to a given checkpoint
-message GetTrialSourceInfoMetricsByCheckpointResponse {
+message GetTrialMetricsBySourceInfoCheckpointResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "data" ] }
+  };
+  // All the related trials and their metrics
+  repeated TrialSourceInfoMetric data = 1;
+}
+
+// Request for all metrics related to a given checkpoint
+message GetTrialSourceInfoMetricsByModelVersionRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "checkpoint_uuid" ] }
+  };
+  // The database id of the model version
+  int32 model_version_id = 1;
```

Comment:
hmmm, isn't this a model_id? not model_version_id.

Reply:
Changed this to better reflect where the id's are actually coming from. It's now `model_id` and `model_version` where they point to the `Model.Id` and `ModelVersion.Version` respectively